infra/general/iniUtils.php

```diff
@@ -7,21 +7,35 @@ class iniUtils
 {
 	/**
 	 * Given an associative array, this function will generate INI file string that represent it.
-	 * @param array $iniData
+	 * @param $ini
+	 * @param bool $isBaseLevel
+	 * @param null $baseKey
 	 * @return string
 	 */
-	public static function arrayToIniString(array $iniData)
+	public static function arrayToIniString($ini, $isBaseLevel = true, $baseKey = null)
```

Comment:
Must run a test vector on this function using all ini's that been called by the system today and make sure we don't break in spacial cases